packages/server/package.json

```diff
@@ -10,6 +10,9 @@
     "url": "git+https://github.com/trpc/trpc.git",
     "directory": "packages/server"
   },
+  "engines": {
+    "node": ">=14"
```

Comment:
Most users won't be affected by this but it is technically a breaking change, is the plan to do a major release?

Reply:
I actually didn't plan to - I never intended to support node `12.x` in the first place and that tsconfig was accidental. 

That said, you're right, it is a major change. I'm going to check if the lib currently even works in node 12 and then decide what to do.

Reply:
Unfortunately, trpc seems to work with `12.x` which means some people might be using it so I might just merge #886 and wait to do this change until next major